pkg/operator/ceph/csi/csi.go

```diff
@@ -238,6 +238,39 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 		}
 	}
 
+	leaderElectionLeaseDuration := k8sutil.GetValue(r.opConfig.Parameters, "CSI_LEADER_ELECTION_LEASE_DURATION", "")
```

Comment:
should we add a upper and lower limit to them?

Reply:
correct, I just wanted to make sure that the user won't add anything that makes the cluster down for a long time and not something less so the cluster cannot select the leader,
Btw would like to pass any doc from CSI that explains how this leader is decided.

Reply:
This is advance setting and its time.Duration flag, if invalid values are passed csi provisioner pod will be down it wont affect the whole cluster or any existing workloads and i assume its admin's responsibility to choose the correct value. https://kubernetes-csi.github.io/docs/introduction.html contains all the docs.